integration-tests/bats/sql-server.bats

```diff
@@ -1873,3 +1873,13 @@ s.close()
     [ $status -eq 0 ]
     [[ "$output" =~ "$EXPECTED" ]] || false
 }
+
+@test "sql-server: binary literal is printed as hex string for utf8 charset result set" {
+    cd repo1
+    start_sql_server
```

Comment:
Does this need to be in server mode? Could the same thing be accomplished via `dolt sql`?

Reply:
The error cannot be produced via `dolt sql`, so keeping it as sql-server test.